app/code/Magento/InventorySalesFrontendUi/composer.json

```diff
@@ -5,6 +5,7 @@
         "php": "~7.1.3||~7.2.0||~7.3.0",
         "magento/framework": "*",
         "magento/module-catalog-inventory": "*",
+        "magento/module-inventory-catalog-frontend-ui": "*",
```

Comment:
This might not be the best showcase of modularity to have Frontend modules depend on other frontend modules